src/test/fuzz/process_message.cpp

```diff
@@ -62,6 +63,11 @@ FUZZ_TARGET(process_message, .init = initialize_process_message)
 {
     FuzzedDataProvider fuzzed_data_provider(buffer.data(), buffer.size());
 
+    g_mock_get_rand = [&](uint64_t max) -> uint64_t {
+        if (fuzzed_data_provider.remaining_bytes() == 0) return 1;
```

Comment:
Not having this exception leads to an endless loop here: https://github.com/bitcoin/bitcoin/blob/c9f288244b8d183e09a917025922b99e3368ef78/src/net_processing.cpp#L5225-L5227